yourgov/components/FormMobileFoodVendorPermit/task/FormTaskFormState.ts

```diff
@@ -11,7 +11,7 @@ import {
 } from '../../../lib/zodUtils';
 import { type Completion } from '../FormState';
 
-export const task1Step1FormSchema = z.object({
+export const taskStep1FormSchema = z.object({
```

Comment:
I feel we should remove the `task` prefix. If we later on want to add multiple tasks, these all need editing anyway. What do you think?